src/components/DataView/DataViewTr/DataViewTrTypes.ts

```diff
@@ -0,0 +1,22 @@
+
+import { MosaicObject } from "@root/types";
+import {
+	DataViewAction,
+	DataViewAdditionalAction,
+	DataViewBulkAction,
+	DataViewColumn,
+	DataViewProps
+} from "../DataViewTypes";
+
+export interface DataViewTrProps {
+	bulkActions?: DataViewBulkAction[];
```

Comment:
Most of these are already declared in DataViewProps (just like you have in the "onReorder") 
```suggestion
	bulkActions?: DataViewProps["bulkActions"];
	checked?: boolean;
	onReorder?: DataViewProps["onReorder"];
	onCheckboxClick?: React.MouseEventHandler<HTMLButtonElement>;
	primaryActions?: DataViewProps["primaryActions"];
	additionalActions?: DataViewProps["additionalActions"];
	originalRowData?: MosaicObject;
	columns?: DataViewProps["columns"];
	row?: MosaicObject;
	rowIdx: number;
```